### ISSUE TITLE
Multiple classes present in the same image

### ISSUE THREAD

**Issue by LewsTherin511** [EscVM/OIDv4_ToolKit#67]
Hi,
I have a problem regarding labeling of images containing instances from multiple classes I want to detect.

Say that I want to train a model to detect "guitars" and "human faces" in pictures.
I download images specifying the 2 classes, and I got (say 1K) images for each class, with the corresponding annotations for bounding boxes.
The problem is that, for every folder of images, only one class is annotated.

So, in the "human face" folder, I have 1000 images, and faces are labelled in every picture.
On the other hand, in the "guitar" folder, I have 1000 images with labelled guitars...however, as you can imagine, in most of the pictures of guitars there are also people, but their faces are not annotated.

At this point, I can't use them for training as they are, because all the faces in the "guitar" images are not labeled as such, and it would provide lots of false negatives to the model.
I tried using the multiclasses argument:

`python3 main.py downloader --classes Guitar Human_face --type_csv train --limit 1000 --multiclasses 1`

and it's the same, the images and annotations are downloaded in the same folder, but they are always separated per class.

Am I doing anything wrong here? Is there a way to download images with labels for all the classes I'm interested in?

**Comment by ahmadalzoubi13579** [comment]
same issue .

**Comment by aishwarya2905** [comment]
I am having the same problem.

**Comment by ufukasia** [comment]
yes this problem valid all multiclass detection.  example i want to detect human head and person but i cant download two box label in one image. this is problem !!!
someone respond this  issue please !!!

**Comment by ahmadalzoubi13579** [comment]
@ufukasia in my case i downloaded annotations files [open image dataset](https://storage.googleapis.com/openimages/web/download.html) (which have detailed information about every image ex: classes) from open image website and then create script to fix every thing, for example taking the image name from downloaded images and search in the annotation file for all classes exist in that image and select the classes i am interesting in.

**Comment by Anas-Alshaghouri** [comment]
Still no solution for this problem?!
COCO has this option but its classes are limited to 80 :(

**Comment by HHendy69** [comment]
I also have the same problem, I want to have the labels of Window and Window blind just as they show up; However the tool shows only one of them. Fiftyone handles that better but needs to adapted to yolo
